src/legacy/core_plugins/input_control_vis/public/control/list_control_factory.js

```diff
@@ -139,6 +140,7 @@ class ListControl extends Control {
       return;
     }
 
+    this.terminatedEarly = resp.terminated_early;
```

Comment:
You will also need to check `timed_out ` in the response

<img width="300" alt="Screen Shot 2019-05-31 at 7 40 09 AM" src="https://user-images.githubusercontent.com/373691/58709465-7b085000-8377-11e9-94c6-47f375f17a15.png">

Reply:
Makes sense, I changed this to `partialResults = resp.terminated_early | resp.timed_out`